Assets/Scripts/StandardSamples/Discord/DiscordManager.cs

```diff
@@ -121,13 +123,54 @@ private class TokenReader
         {
             public string access_token;
             public float expires_in;
+            public string refresh_token;
         }
 
         private string cachedToken = null;
+        private string refresh_token = null;
         private DateTime tokenExpiration;
 
         private event Action<string> onTokenReceived;
 
+        public void StartConnectLoginWithDiscord(EOSManager.OnConnectLoginCallback onLoginCallback)
+        {
+            StartConnectLoginWithDiscord(onLoginCallback, false);
+        }
+
+        private void StartConnectLoginWithDiscord(EOSManager.OnConnectLoginCallback onLoginCallback, bool isRetry = false)
```

Comment:
Out of curiosity (and depending on the answer this may be a critique), is there a reason to use the suffix `WithDiscord`, given the context that these functions are defined within the `DiscordManager`? Isn't `WithDiscord` therefore an extraneous descriptor? My thought process is that if this is the case, then reducing the function name to exclude the suffix might make it helpfully align with existing interfaces.

Reply:
Given the location of this function, there is no need for it to be "WithDiscord". Resolving by removing as suggested.